apiserver/plane/bgtasks/export_task.py

```diff
@@ -307,51 +312,116 @@ def issue_export_task(provider, workspace_id, project_ids, token_id, multiple, s
-                    "created_by__first_name",
-                    "created_by__last_name",
-                    "assignees__first_name",
-                    "assignees__last_name",
-                    "labels__name",
-                )
+            Issue.objects.filter(
+                workspace__id=workspace_id,
+                project_id__in=project_ids,
+                project__project_projectmember__member=exporter_instance.initiated_by_id,
+                project__project_projectmember__is_active=True,
+                project__archived_at__isnull=True,
+            )
+            .select_related(
+                "project",
+                "workspace",
+                "state",
+                "parent",
+                "created_by",
+                "estimate_point",
+            )
+            .prefetch_related(
+                "labels",
+                "issue_cycle__cycle",
+                "issue_module__module",
+                "issue_comments",
+                "assignees",
+                "issue_subscribers",
+                "issue_link",
             )
-            .order_by("project__identifier", "sequence_id")
-            .distinct()
         )
-        # CSV header
+
+        file_assets = FileAsset.objects.filter(
+            issue_id__in=workspace_issues.values_list("id", flat=True)
+        ).annotate(work_item_id=F("issue_id"), asset_id=F("id"))
+
+        attachment_dict = defaultdict(list)
+        for asset in file_assets:
+            attachment_dict[asset.work_item_id].append(asset.asset_id)
+
+        issues_data = []
+
+        for issue in workspace_issues:
+            attachments = attachment_dict.get(issue.id, [])
+
+            issue_data = {
+                "id": issue.id,
+                "project_identifier": issue.project.identifier,
+                "project_name": issue.project.name,
+                "project_id": issue.project.id,
+                "sequence_id": issue.sequence_id,
+                "name": issue.name,
+                "description": issue.description_stripped,
+                "priority": issue.priority,
+                "start_date": issue.start_date,
+                "target_date": issue.target_date,
+                "state_name": issue.state.name if issue.state else None,
+                "created_at": issue.created_at,
+                "updated_at": issue.updated_at,
+                "completed_at": issue.completed_at,
+                "archived_at": issue.archived_at,
+                "module_name": [
+                    module.module.name for module in issue.issue_module.all()
+                ]
+                if issue.issue_module.exists()
+                else None,
+                "created_by": f"{issue.created_by.first_name} {issue.created_by.last_name}",
+                "labels": [
+                    label.name
+                    for label in issue.labels.all().distinct()
+                    if issue.labels.exists()
```

Comment:
@sangeethailango The `exists()` an `distinct()` will make separate queries to db. Update the logic to avoid that.